daemon/graphdriver/aufs/aufs.go

```diff
@@ -76,8 +77,9 @@ type Driver struct {
 	root       string
 	uidMaps    []idtools.IDMap
 	gidMaps    []idtools.IDMap
-	sync.Mutex // Protects concurrent modification to active
+	globalLock sync.Mutex // Protects concurrent modification to active
 	active     map[string]*data
+	idLock     *locker.Locker
```

Comment:
Can we get a comment that this must be locked before locking the globalLock? You were very careful to use that order, so it would be nice to document it.